### PR TITLE
Intro logo and other branding/naming changes

### PR DESCRIPTION
**Pull request by andybak**

### CODE REVIEW

**Comment by mikeage** [comment]
Looks fine to me, but I have no way whatsoever to test it, so... I dunno.

Are all of the files added (like the font from the logo) things that belong in this repo, or are they here because we have nowhere else to put them? They're pretty tiny, so I don't really care either way, but just curious.

**Comment by mikeage** [comment]
Also, I see references to TMP now; do we need to do the same thing we did in Open Brush to fully install the TextMeshPro package?

File: ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -168,8 +169,8 @@ PlayerSettings:
   androidSupportedAspectRatio: 1
   androidMaxAspectRatio: 2.1
   applicationIdentifier:
-    Android: com.oculus.UnitySample
-    Standalone: unity.Google.Blocks
```
Comment (mikeskydev) [inline ProjectSettings/ProjectSettings.asset:172]: foundation.icosa.openblocks :)

Comment (andybak) [reply ProjectSettings/ProjectSettings.asset:172]: fixed

**Comment by andybak** [comment]
> Also, I see references to TMP now; do we need to do the same thing we did in Open Brush to fully install the TextMeshPro package?

It's currently .gitignored (which I probably copied from Open Brush - but any reason not to just add it to the repo? It's only 12mb and it never changes.

Seems like a simpler and less brittle solution.

**Comment by mikeage** [comment]
I think the Open Brush version was a combination of license considerations and the fact that they didn't just have a regular package, but this installation method that had to be run, and I wanted to be sure that it did everything it needed to.

It looks brittle, but it's also lasted perfectly for years, so...

**Pull request merged**